locale/he/locale.cfg

```diff
@@ -24,14 +24,25 @@ sensor.player_locations.err_no_name=.םיפתתשמ הבורמ קחשמ לרטנ
 sensor.player_locations.surface_fragment=__1__ לע
 sensor.player_locations.offline_fragment=]קתונמ[
 sensor.player_locations.settings.title=ןקחש ימוקימ תורדגה
-sensor.player_locations.settings.show_player_index=ןקחש רפסמ הארה
-sensor.player_locations.settings.show_position=ןויצ תודוקנ הארה
-sensor.player_locations.settings.show_surface=וילע ןקחשהש םוקמה תא הארה
-sensor.player_locations.settings.show_direction=ןקחשל ןוויכ הארה
+sensor.player_locations.settings.show_player_index=ןקחש רפסמ
+sensor.player_locations.settings.show_position=ןויצ תודוקנ
+sensor.player_locations.settings.show_surface=וילע ןקחשהש םוקמה תא
+sensor.player_locations.settings.show_direction=ןקחשל ןוויכ
+sensor.player_locations.settings.show_offline=םיקתונמ םינקחש
 
 sensor.pollution_around_player.name=הבורקה הביבסב םוהיז תמר
 sensor.pollution_around_player.format=PU. __1__ :םוהיז תמר
 
+sensor.kill_count.name=(הצובקל) תוגירה
+sensor.kill_count.format=Kills: __1__, __2__, __3__.
```

Comment:
My mistake - didn't see that one - It got lost with all the other text and underscore line & my a.d.d. also helps (interruptions from my job...)